src/packages/overlay/overlay.tsx

```
@@ -48,61 +38,39 @@
-
   const classes = classNames(classPrefix, className)
-
   const styles = {
     ...style,
+    zIndex,
   }
 
-  const handleClick: MouseEventHandler<HTMLDivElement> = (e: MouseEvent) => {
+  const handleClick = (e: MouseEvent) => {
     if (closeOnOverlayClick) {
       onClick && onClick(e)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)